test/syncenginetestutils.h

```diff
@@ -127,6 +127,15 @@ class FileInfo : public FileModifier
     FileInfo(const QString &name, qint64 size, char contentChar, QDateTime mtime) : name{name}, isDir{false}, lastModified(mtime), size{size}, contentChar{contentChar} { }
     FileInfo(const QString &name, const std::initializer_list<FileInfo> &children);
 
+    enum Etags {
+        Validate = 0,
```

Comment:
```suggestion
        KeepUnmodified = 0,
```
feel free to find a better name
it is either the value is invalidated or the previous value is kept